a11y.js

```diff
@@ -0,0 +1,19 @@
+const pageWidths = [320, 400, 600, 1008, 1280];
```

Comment:
Can this file be moved out of the root directory?

Reply:
Not according to the docs of bbca11y. 

Also re 'Is there any reason to not use https://github.com/bbc/a11y-tests-web?' 1. the issue says to use this, but I evaluated it anyway, see https://github.com/bbc/simorgh/issues/331 and I've opened PRs on that repo to clarify that docker isn't needed. Aside from extending the functionality the setup is unclear and I wanted to tackle using it along with setup in jenkins so I can contribute back.